docs/webhooks/create-incoming-webhook.md

```diff
@@ -61,6 +61,24 @@ Webhooks, for now, are an API only feature; so this post will guide you through
 			    	"type": "integer",
 				    "required": false,
     				"default_value": <pipeline_id>
+	    		},
```

Comment:
Could you please set the `required_permissions` values for the operations described in this post?

Reply:
Done

Reply:
1) Suggestion to be more specific on the footer note

From:
```
For instructions on how to check this information, see
```

To

```
For instructions on how to grant user permissions in an environment, see
```

Because we're linking to a topic to ensure user success on granting permission, and not only `"to check"`.
Also we should avoid `this information` and be as specific as possible, IMHO.

<img width="816" alt="image" src="https://github.com/devopness/devopness/assets/1943318/9d1dbe7e-66dc-4082-adaf-77861d0dbb42">

Reply:
2) `/docs/environments/add-teams-to-environment/` has a broken link on
```
please follow the guide Check Role Permissions
```

Reply:
1. About required permissions footnotes

https://github.com/devopness/devopness-site/pull/9

Reply:
2. Broken url Check Role Permission

Fixed